app/models/practice_editor.rb

```diff
@@ -25,4 +25,13 @@ def ensure_practice_has_at_least_one_practice_editor
       throw :abort
     end
   end
-end
\ No newline at end of file
+
+  def self.get_editor_practices(user)
+    editor_list = PracticeEditor.where(user: user)
+    practices = []
+    editor_list.each do |e|
+      practices.push(Practice.find(e.practice_id))
+    end
+    practices
+  end
+end
```

Comment:
Are we using this? I didn't see any references to it, that's why I'm asking.

Reply:
haha nope! Nice catch I just deleted it now :)